vendor/github.com/openshift/api/config/v1/0000_00_cluster-version-operator_01_clusteroperator.crd.yaml

```diff
@@ -0,0 +1,162 @@
+                description: ClusterOperatorStatusCondition represents the state of
+                  the operator's reconciliation functionality.
+                type: object
+                required:
+                - lastTransitionTime
+                - status
+                - type
+                properties:
+                  lastTransitionTime:
+                    description: lastTransitionTime is the time of the last update
+                      to the current status object.
+                    type: string
+                    format: date-time
+                  message:
+                    description: message provides additional information about the
+                      current condition. This is only to be consumed by humans.
+                    type: string
+                  reason:
+                    description: reason is the reason for the condition's last transition.  Reasons
+                      are CamelCase
+                    type: string
+                  status:
+                    description: status of the condition, one of True, False, Unknown.
+                    type: string
+                  type:
+                    description: type specifies the state of the operator's reconciliation
+                      functionality.
+                    type: string
+            extension:
+              description: extension contains any additional status information specific
+                to the operator which owns this status object.
+              type: object
+              nullable: true
+            relatedObjects:
+              description: 'relatedObjects is a list of objects that are "interesting"
+                or related to this operator.  Common uses are: 1. the detailed resource
+                driving the operator 2. operator namespaces 3. operand namespaces'
+              type: array
+              items:
+                description: ObjectReference contains enough information to let you
+                  inspect or modify the referred object.
+                type: object
+                required:
+                - group
+                - name
+                - resource
+                properties:
+                  group:
+                    description: group of the referent.
+                    type: string
+                  name:
+                    description: name of the referent.
+                    type: string
+                  namespace:
+                    description: namespace of the referent.
+                    type: string
+                  resource:
+                    description: resource of the referent.
+                    type: string
+            versions:
+              description: versions is a slice of operand version tuples.  Operators
+                which manage multiple operands will have multiple entries in the array.  If
+                an operator is Available, it must have at least one entry.  You must
```

Comment:
Even operators with a single operand will have multiple entries, one for `operator` and one for the operand.  And when you're `Available`, you need to have an `operator` entry, not just "an entry".  I've filed openshift/api#501 to wordsmith that sort of thing.